packages/makecode-core/src/commands.ts

```diff
@@ -977,10 +1012,10 @@ async function getExpandedPageFromCache(cacheKey: string, cache: mkc.Cache, webs
         const contentString = await processElement(element, "src", ".js");
 
         if (contentString) {
-            scripts.push(btoa(encodeURIComponent(contentString)));
+            scripts.push(encodeURIComponent(contentString));
```

Comment:
Are the `btoa` and `atob` not needed because you've changed how we're handling encoded strings now?

Reply:
encodeURIComponent is sufficient to escape the scripts in the string literals